src/mumble/MumbleApplication.cpp

```diff
@@ -11,6 +11,54 @@
 #include "GlobalShortcut.h"
 #include "Global.h"
 
+// getenvQString is a wrapper around _wgetenv_s (on Windows)
+// and getenv (on everything else).
+//
+// On Windows, it expects a Unicode environment -- so variables
+// are expected to be UTF16.
+// On everthing else, it expects the environment variables to be
+// UTF-8 encoded.
+static QString getenvQString(QString name) {
+#ifdef Q_OS_WIN
+	QByteArray buf;
+	size_t requiredSize = 0;
+
+	static_assert(sizeof(wchar_t) == sizeof(ushort), "expected 2-byte wchar_t");
+
+	const wchar_t *wname = reinterpret_cast<const wchar_t *>(name.utf16());
+
+	// Query the required buffer size (in elements).
+	_wgetenv_s(&requiredSize, 0, 0, wname);
+	if (requiredSize == 0) {
+		return QString();
+	}
+
+	// Resize buf to fit the value and put it there.
+	buf.resize(static_cast<int>(requiredSize * sizeof(wchar_t)));
+	_wgetenv_s(&requiredSize, reinterpret_cast<wchar_t *>(buf.data()), requiredSize * sizeof(wchar_t), wname);
```

Comment:
Totally missed that. Yeah. That should be fixed (even though it can't really break anything outside of a race condition).